clang/test/CIR/CodeGen/builtin_call.cpp

```diff
@@ -76,3 +76,21 @@ float constant_fp_builtin_single() {
 // OGCG: define {{.*}}float @_Z26constant_fp_builtin_singlev()
 // OGCG: ret float 0x3FB99999A0000000
 // OGCG: }
+
+void library_builtins() {
+  __builtin_printf(nullptr);
```

Comment:
Will `__builtin_printf` also work with format and data arguments? Can you add a test case for that?

Reply:
I added `builtin_printf.cpp` that contains a few `printf` calls